CONTRIBUTING.md

```diff
@@ -1,73 +1,76 @@
+* Call ./scripts/format.sh before pushing.
+* Choose clear and concise names, and avoid, e.g., `i`, `j`, `ch_ptr`.
+* Formatting details: 2 spaces, 120 columns, comments above code.
+* Use empty lines to structure your code.
+* Naming conventions:
+
+    * Files: lowercase separated by underscores, e.g., abstract_operator.cpp, usually corresponding to a class, e.g., AbstractOperator.
+
+    * Types (classes, structs, enums, typedefs, using): CamelCase starting with uppercase letter, e.g., `TableScan`.
+
+    * Variables: lowercase separated by underscores, e.g., `chunk_size`.
+
+    * Functions: lowercase separated by underscores, e.g., `append_mutable_chunk()`.
+
+    * Private / protected members / methods: like variables / functions with leading underscore, e.g., `_on_execute()`.
+
+    * Classes that are used only to have a non-templated base class are named `BaseXY` (e.g., `BaseValueSegment`, while classes that have multiple differing implementations are named `AbstractXY` (e.g., `AbstractOperator`).
+
+    * In cases where a constructor parameter would have the same name as the member it initializes, prefix it with `init`: `C(int init_foo) : foo(init_foo) {}`.
+
+    * If an identifier contains a verb or an adjective in addition to a noun, the schema [verb|adjective]\[noun] is preferred, e.g., use `left_input` rather than ~~`input_left`~~ and `set_left_input()` rather than ~~`set_input_left()`~~.
+
+* Maintain correct orthography and grammar. Comments should start with a capital letter, sentences should be finished with a full stop.
+  * Class names within comments are written in CamelCase - e.g., "As the input may be a ReferenceSegment, a valid RowID may *point to* a row that is NULL."
+
+# Pull Requests
+## Opening PRs
+* When you submit a non-trivial PR, include the results of benchmark_all.sh.
+  * These help in understanding potential performance changes as well as document potential changes to the compilation costs.
+  * We do not do this automatically as the CI server is not sufficiently isolated and the performance results would vary. Similarly, your personal laptop is likely to produce unreliable results.
+* If your PR is related to an existing issue, reference it in the PR's description (e.g., `fixes #123`).
+* If you are not a member of the Hyrise organization, your PR will not be built by our CI server. Contact a maintainer for this. They can add you to the organization or manually trigger builds from within Jenkins.
+* When merging your PR, copy your PR description (excluding the benchmark results) into the commit message. The commit message of the squash merge should NOT contain the individual commit messages from your branch.
+* For your PR to be merged, it must pass the FullCI run. Set the FullCI tag in Github before committing to trigger the complete (but more expensive!) run.
+
+## Reviewing PRs
```

Comment:
PRs > Pull Requests

Reply:
Haven't seen the title above where we write "Pull Request". Maybe just drop the "PRs"?

Reply:
Not sure what your point is. We use "PR" in every second line here.

Reply:
I don't like unnecessary abbr in titles. That's all. But that's probably just me. Resolved.